### DISCUSSION TITLE
An error occurred while installing dependencies

### DISCUSSION THREAD

**Discussion by savifon** [storybookjs/storybook#18008] [Help]
This error is occurring after running the npx sb init command to install Storybook in my Vue 3 (Vite with TS) application:

```
$ npx sb init

 sb init - the simplest way to add a Storybook to your project. 

 • Detecting project type. ✓
 • Adding Storybook support to your "Vue 3" app. ✓
 • Preparing to install dependencies. ✓


 • Installing dependencies. ✖

     An error occurred while installing dependencies.
```

`npx sb init --type vue3` doesn't work either

How to adjust to install correctly?

**Comment by bioudi** [comment]
Getting same issue any update on this ?

**Reply by bioudi** [reply]
I was able to fix it by re-running **npm i** after getting that error.

**Comment by savifon** [comment]
trying to run the command a few days later, it worked. Could it be some specific configuration

**Comment by bozzelliandrea** [comment]
i can't reproduce or found this problem on my machine, so the question could be answered and set done status.
